### ISSUE TITLE
Need For Speed: Unbound (1846380)

### ISSUE THREAD

**Issue by adro79** [ValveSoftware/Proton#6436]
# Compatibility Report
- Name of the game with compatibility issues: Need For Speed: Unbound
- Steam AppID of the game: 1846380

## System Information
- GPU: RTX 3090
- Driver/LLVM version: nvidia 525.60.11
- Kernel version: 6.1.2 Zen
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: Proton Experimental, Proton G.E., Proton 7.0

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
DirectX error when launching story mode. (Online mode has no issues)
This started ocurring 2 days ago.

I tried reinstalling the game, playing in windowed mode and lowering the resolution.

PD: I abstain to upload the file log since it seems that produces a 1GB log. (Normal?)

```
DirectX function
"Dx12Renderer::waitOnSwapChain Gpu timeout
detected" failed with DXGI_ERROR_DEVICE_HUNG
("The GPU will not respond to more comands").
GPU: "NVIDIA GeForce RTX 3090", Driver: 52560.
This error is usually caused by the lastest graphics driver
crashing; try installing the lastest drivers.
GPUBreadcrumbs:
```

## Reproduction
1. Start NFS
2. Open Story Mode
3. Crash Pop-up appears.

**Comment by adro79** [comment]
Also the game freezes when I try to enable FSR or DLSS and changing the quality of the graphics.

**Comment by adro79** [comment]
The issue got fixed in the lastest nvidia update 525.78.01

**Comment by Negrumon** [comment]
when I try to open the game (need for speed unbound), a error shows up to me. How do I fix this? my game is cracked

<img width="602" height="279" alt="Image" src="https://github.com/user-attachments/assets/8405d18d-b277-4c9f-9abc-972b38dbfa7d" />
